gluster-exporter/metric_volume.go

```diff
@@ -173,8 +175,132 @@ var (
 		LongHelp:  "",
 		Labels:    volumeProfileFopInfoLabels,
 	})
+
+	glusterVolumeProfileFopTotalHitsAggregatedOps = newPrometheusGaugeVec(Metric{
+		Namespace: "gluster",
+		Name:      "volume_profile_fop_total_hits_on_aggregated_fops",
```

Comment:
Make this name to represent that this is cumulative metrics as doing at line#190

Reply:
As discussed, currently used the existing naming conventions, followed in the existing code, and probably, at a later point, we can do a refactoring...